slsim/Deflectors/richness2mass.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+
+
+def general_mass_richness(rich, alpha, rich_0, m_0, ln_m_scatter, min_mass=0.0):
+    """M(N) = M_0 * (N / N_0) ** alpha VAR[ln(M)] = alpha**2 / N + lnM_scatter**2."""
```

Comment:
Missing documents for the input parameters.

Reply:
added documentation